src/hotspot/share/gc/shenandoah/heuristics/shenandoahOldHeuristics.cpp

```diff
@@ -192,7 +173,69 @@ bool ShenandoahOldHeuristics::prime_collection_set(ShenandoahCollectionSet* coll
     collected_old_bytes += r->garbage();
     consume_old_collection_candidate();
   }
+  return true;
+}
+
+void ShenandoahOldHeuristics::initialize_piggyback_evacs(ShenandoahCollectionSet* collection_set,
```

Comment:
Can we continue using `mixed` instead of `piggyback` to describe collections that include young and old regions? I think `mixed` is an accepted term and `piggyback` feels a little bit too colloquial (also not sure if the phrase is commonly known outside of English).

Reply:
Good suggestion.  Thanks.  I'm replacing piggyback with mixed throughout (where piggyback means mixed evac).  There are a few places where piggyback in Shenandoah where piggyback is used for other purposes and I'm leaving those as is.